bundles/Eiffel/docker-compose.yml

```diff
@@ -29,7 +29,7 @@
 # Author: michael.frick@ericsson.com
 #
 ##################################################################################################
-version: "3.7"
```

Comment:
There is one more docker-compose file [here.](https://github.com/eiffel-community/eiffel-easy2use/blob/master/bundles/Eiffel/build-source/docker-compose.yml) Do we need to change its "version" as well?

Reply:
That docker-compose is not used anymore, Dockerfiels and build is done in each Eiffel GitHub project soruce code repository. And I haven't tested that docker-compose build and I think those builds are not working anymore.
Maybe should be removed instead?

Reply:
Would be good to remove code if we don't need it!

Reply:
When I check it again, it seems to be used for build the jenkins image with Eiffel plugin, and two more docker images.
So maybe should be kept if these images need to be re-built with newer Jenkins version etc.
But since I have tested the builds with newer docker-compose specification version, I will not update the version. Maybe someone else can do that in a separate patch.